src/Admin/AdminHelper.php

```diff
@@ -14,6 +14,7 @@
 namespace Sonata\AdminBundle\Admin;
 
 use Doctrine\Common\Collections\Collection;
+use Doctrine\Common\Inflector\Inflector;
```

Comment:
I did not check, is this the correct one or the deprecated? @greg0ire do you know?

Reply:
I think it's the deprecated one: https://github.com/doctrine/inflector/blob/2.0.x/lib/Doctrine/Inflector/Inflector.php

Reply:
Can you use the not deprecated? @peter-gribanov

Reply:
Ok. I changed it